### PR TITLE
Added profile for SecKit SK-Tank

### PR DESCRIPTION
**Pull request by scottmudge**
Added profile for SK-Tank (https://sites.google.com/view/seckit-wiki/sk-tank-350x350x400) CoreXY printer.

Some notes:

* Default max acceleration set to just above the recommended used in the klipper configs linked in the SecKit SK-Tank wiki.
* Added custom build-plate STL and logo. 
* I could not find any good "stock" image of the printer alone for the setup wizard image for the printer, so I used the SecKit logo instead. All of the images in the Wiki did not look good when I erased the background, since there is a laptop stuck on the top of the machine.

### CODE REVIEW

File: resources/web/image/printer/SecKit SK-Tank_cover.png
Comment (SoftFever) [inline resources/web/image/printer/SecKit SK-Tank_cover.png:1]: Can you put a printer image instead?

File: resources/profiles/SecKit/hotend.stl
Comment (SoftFever) [inline resources/profiles/SecKit/hotend.stl:1]: we don't need this hotend.stl anymore

**Review by SoftFever — CHANGES_REQUESTED** [review]

File: resources/web/image/printer/SecKit SK-Tank_cover.png
Comment (scottmudge) [reply resources/web/image/printer/SecKit SK-Tank_cover.png:1]: Yeah I'll have to work some photoshop wizardry, but I'll see what I can do.

File: resources/profiles/SecKit/hotend.stl
Comment (scottmudge) [reply resources/profiles/SecKit/hotend.stl:1]: Good to know -- I didn't see it really used anywhere, but the other profiles had it, so followed convention. I'll delete it.

File: resources/web/image/printer/SecKit SK-Tank_cover.png
Comment (scottmudge) [reply resources/web/image/printer/SecKit SK-Tank_cover.png:1]: Alright, took a lot of using the eraser tool and clone-stamp tool, but I created an image from the SecKit website: https://seckit3dp.design/img/sk-tank-main.webp

File: resources/profiles/SecKit/hotend.stl
Comment (scottmudge) [reply resources/profiles/SecKit/hotend.stl:1]: It's been removed

**Comment by scottmudge** [comment]
Thanks for the review -- changes have been completed and PR updated.

**Review by SoftFever — APPROVED** [review]
LGTM.
Thank you

**Pull request merged**